### PR TITLE
Docker images for CGOS

### PR DESCRIPTION
**Pull request by amj**
we can build off this to make some images for fuego/pachi/etc

### CODE REVIEW

File: cc/gtp_player.cc [hunk 1]
```diff
@@ -312,6 +313,20 @@ GtpPlayer::Response GtpPlayer::HandleInfo(
   return Response::Ok(oss.str());
 }
 
+GtpPlayer::Response GtpPlayer::HandleKomiCommand(
```
Comment (tommadams) [inline cc/gtp_player.cc]: ordering nit: Komi comes after Known

File: cc/gtp_player.h [hunk 2]
```diff
@@ -110,6 +110,9 @@ class GtpPlayer : public MctsPlayer {
   Response HandleInfo(absl::string_view cmd,
                       const std::vector<absl::string_view>& args);
 
+  Response HandleKomiCommand(absl::string_view cmd,
```
Comment (tommadams) [inline cc/gtp_player.h]: same here

File: cc/gtp_player.cc (hunk 1 above)
Comment (amj) [reply cc/gtp_player.cc]: err, right...

**Comment by cjwagner** [comment]
/test pull-tf-minigo-cc
(should report a status now)

**Comment by amj** [comment]
whoa!  @cjwagner that looks like a real failure -- anyway to see the test failure in the stuff at the "details" link?

**Comment by cjwagner** [comment]
Yeah, bazel doesn't output logs for failed tests by default 😞 
I made https://github.com/kubernetes/test-infra/pull/8216 to fix this.

**Comment by cjwagner** [comment]
The test passed this time so it must be flaky.
I'll run it again to see if we can get a failure now that the logs will be useful.
/test pull-tf-minigo-cc

**Comment by amj** [comment]
woohoo!  looks like tom tracked down the flaky test in #255 .  Merging!

**Pull request merged**